tests/Behat/OAuthContext/OAuthContext.php

```
@@ -99,36 +110,57 @@ public function passingIdSecretEmailAndPassword(
     }
 
     /**
-     * @Given client with id :id, secret :secret and redirect uri :uri exists
+     * @Given client with id :id, secret :secret and redirect_uri :uri exists
      */
     public function clientExists(string $id, string $secret, string $uri): void
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (Naming Rules)

(ShortVariable)